linode/kernels/tmpl/data_filter.gotf

```diff
@@ -0,0 +1,12 @@
+{{ define "kernels_data_filter" }}
+
+{{ template "kernels_data_base" . }}
+
+data "linode_kernels" "kernels" {
+    filter {
```

Comment:
Could additional filters be added here for each filterable fields?

Reply:
Yes, done!